converters/fromZigbee.js

```diff
@@ -1083,8 +1091,38 @@ const converters = {
             };
         },
     },
-
+    thermostat_devChange: {
+        cid: 'hvacThermostat',
+        type: 'devChange',
+        convert: (model, msg, publish, options) => {
+            return {
+                localTemp: precisionRound(msg.data.data['localTemp'], 2) / 100,
+                occupiedHeatingSetpoint: precisionRound(msg.data.data['occupiedHeatingSetpoint'], 2) / 100,
+                setpointChangeSource: msg.data.data['setpointChangeSource'],
+                setpointChangeAmount: msg.data.data['setpointChangeAmount'] / 100,
+                setpointChangeSourceTimeStamp: msg.data.data['setpointChangeSourceTimeStamp'],
```

Comment:
Please change to: local_temperature, occupied_heating_setpoint, setpoint_change_source, setpoint_change_amount, setpoint_change_source_timestamp